src/FINDOLOGIC/Api/Definitions/QueryParameter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace FINDOLOGIC\Api\Definitions;
+
+class QueryParameter
+{
+    const
+        SHOPKEY = 'shopkey',
```

Comment:
I suggest changing this to `SERVICE_ID`, because this term is the successor of `SHOPKEY`.